openassessment/xblock/static/js/src/lms/oa_staff.js

```diff
@@ -0,0 +1,34 @@
+/**
+Interface for staff assessment view.
```

Comment:
I thought that JSCS would validate JSDoc headers but it doesn't seem to be failing.